Nancy.Rollbar/RollbarDataFactory.cs

```diff
@@ -78,8 +78,10 @@ public class RollbarDataFactory : IRollbarDataFactory {
                     .ToDictionary(x => x.Key, x => x.Value);
                 if (rr.PostParams.Count == 0) {
                     rr.PostParams = null;
-                }
-                rr.PostBody = context.Request.Body.AsString();
+                }
+                if (!context.Request.Body.CanRead) {
+                    rr.PostBody = context.Request.Body.AsString();

```

Comment:
only if you don't check if its empty or null. (most likely the only library issue would be in the scrubber, which isn't implemented in this library)

We could fill it with an empty string....but I feel as though leaving it as null signifies all did not go as planned.